qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -1216,6 +1217,8 @@ def _uhrig_single_axis_offsets(
     numpy.ndarray
         The offset values
     """
+    # in case a float number is passed
+    number_of_offsets = int(number_of_offsets)
```

Comment:
I'm fine with this since it's consistent with the other functions, but in general I prefer to do this sort of input-sanitisation as close as possible to where the user provides it. That way you limit the amount of code that can be exposed to the raw (possibly-ridiculous) user input.